setup.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup
 
-version = '0.8.9'
-homepage = 'https://github.com/python-amazon-mws/python-amazon-mws'
+version = '0.8.10'
+homepage = 'http://github.com/python-amazon-mws/python-amazon-mws'
```

Comment:
should be https (be good on the web and all that!)

however, there is a redirect from http -> https so its not a show stopper.